src/contracts/core/DelegationManager.sol

```diff
@@ -682,6 +683,11 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
                         });
                     }
                 } else {
+                    require(
+                        withdrawal.startBlock + strategyWithdrawalDelayBlocks[withdrawal.strategies[i]] <= block.number, 
+                        "DelegationManager.completeQueuedAction: withdrawalDelayBlocks period has not yet passed for this strategy"
```

Comment:
While we're at it, let's change the error text here to `completeQueuedWithdrawal` :sweat: 

The `completeQueuedAction` text is a holdover from my very first chicken scratch refactor of the withdrawal queues

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/392/commits/323fa4e387941179468d541a4c9d9470c2382bd1#diff-d323417ded59a593f4ea410225a9167b24198c4d2c05cbc0fe9af47d34b88810R646